packages/core-app-api/report.api.md

```diff
@@ -294,6 +294,13 @@ export type AuthApiCreateOptions = {
   configApi?: ConfigApi;
 };
 
+// @public
+export type AuthConnector<AuthSession> = {
+  createSession(options: CreateSessionOptions): Promise<AuthSession>;
+  refreshSession(scopes?: Set<string>): Promise<AuthSession>;
```

Comment:
Could you wrap this one up in options as well? So `options?: AuthConnectorRefreshSessionOptions` with `scopes` inside